parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -340,6 +344,18 @@ ensureTeleportd() {
     systemctlEnableAndStart teleportd || exit $ERR_SYSTEMCTL_START_FAIL
 }
 
+ensureArtifactStreaming() {
+  systemctl start acr-mirror.service
+  sudo /opt/acr/tools/overlaybd/install.sh
+  sudo /opt/acr/tools/overlaybd/enable-http-auth.sh
+  modprobe target_core_user
+  curl -X PUT 'localhost:8578/config?ns=_default&enable_suffix=azurecr.io&stream_format=overlaybd'
```

Comment:
do we want to parse the response from the curl request to check for failures or this will always succeed? since I guess the following steps of enabling and starting the 2 systemd units depends on this PUT?

Reply:
Writing the output to stdout so it can be looked at with CSE logs if there are failures. There is a chance this may not succeed. It doesn't give very actionable responses though, so having it in stdout might be sufficient.